src/markdoc/tags/Tabs.svelte

```diff
@@ -26,7 +26,6 @@
 </script>
 
 <div class="aw-card is-normal u-margin-block-start-16" {...$root} use:root>
-	<nav class="tabs u-flex u-gap-16">
```

Comment:
Sorry, my wording was off here. By removing the nav element, I meant swapping it with something else, e.g. a div. Otherwise we lose the styling we had here.